doc/src/learn/single-writer-apps.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Sui Single-Writer-Friendly (SWF) apps
+---
+
+This page lists applications that can work in the single-writer model defined as [simple transactions](how-sui-works.md#simple-transactions) in Sui. Some proposals that typically require shared objects have been transformed to variants that require only a shared object as a final step and not for every action, such as voting and lotteries, DeFi Oracle price quotes, etc.
```

Comment:
Replace "Some proposals that typically..." with
"Apart from the obvious single-writer friendly applications (such as simple peer to peer asset transfer), note that some proposals that typically..." proposals that typically..."